### PR TITLE
LPS-186809

### PR DESCRIPTION
**Pull request by TamyrisTorres**
Ticket: https://issues.liferay.com/browse/LPS-186809

### CODE REVIEW

**Comment by TamyrisTorres** [comment]
ci:test:sf

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeFtlGetPortletGroupIdMethodCheck.testftl [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<#-- FreeMarker Template -->
+
+<#assign
+	group = themeDisplay.getScopeGroupId()
+/>
+<@liferay_aui.col width=25>
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeFtlGetPortletGroupIdMethodCheck.testftl:6]: @TamyrisTorres, can you please remove this taglib from this test?

It's not so necessary considering the test's scope

Thanks

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJspGetPortletGroupIdMethodCheck.testjsp [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+		<c:if test="<%= panelApp.isShow(permissionChecker, themeDisplay.getScopeGroupId()) %>">
+			test
+		</c:if>
+
+	<%
+	}
+	%>
+
+</c:if>
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJspGetPortletGroupIdMethodCheck.testjsp:40]: @TamyrisTorres, please keep the test scope as simple as possible, like the test file related with the Java file (modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaGetPortletGroupIdMethodCheck.testjava)

**Review by natocesarrego — CHANGES_REQUESTED** [review]

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJspGetPortletGroupIdMethodCheck.testjsp [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/panel/init.jsp" %>
+
+	<c:if test="<%= panelApp.isShow(permissionChecker, themeDisplay.getPortletGroupId()) %>">
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJspGetPortletGroupIdMethodCheck.testjsp:19]:
```
%@ include file="/panel/init.jsp" %>

<c:if test="<%= panelApp.isShow(permissionChecker, themeDisplay.getPortletGroupId()) %>">
</c:if>
```

**Pull request closed without merging**